src/components/Todo.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from "react";
+
+import { TodoInterface } from "types/todos";
+
+import { StyleSheet, css } from "aphrodite";
+import { colors, typography } from "styles";
+import { i18n } from "locale";
+
+const Todo: FC<TodoInterface> = ({ id, text }) => {
+  const handleClick = (): void => {
+    alert(i18n.t("test:alertTodo"));
+  };
+  return (
+    <>
+      <div
+        className={css(styles.task)}
+        onClick={() => {
+          handleClick();
+        }}
+        onKeyDown={handleClick}
+        role="menuitem"
+        tabIndex={Math.random()}
+      >
+        <div className={css(styles.id, typography.altBody1)}>
+          <p>{id}</p>
+        </div>
+        <div className={css(styles.text, typography.altBody1)}>
+          <p>{text}</p>
+        </div>
+      </div>
+    </>
+  );
+};
+
+const styles = StyleSheet.create({
+  task: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    display: "flex",
+    overflow: "hidden",
+    transition: "0.5s",
+    margin: "5px auto",
+    boxShadow: `3px 3px 5px ${colors.blue4}`,
+    border: `1px solid ${colors.white}`,
+    borderRadius: 5,
+    maxWidth: "80%",
+    width: "400px",
```

Comment:
I m almost sure that this maxWidth is unnecessary

Reply:
In portrait orientation it looks better with maxWidth.

Reply:
![portrait](https://user-images.githubusercontent.com/83353165/123523791-5bfb5a00-d6c6-11eb-81e9-f96913465831.png)